controllers/designatebackendbind9_controller.go

```diff
@@ -661,6 +662,18 @@ func (r *DesignateBackendbind9Reconciler) reconcileNormal(ctx context.Context, i
 	}
 	// create StatefulSet - end
 
+	// Handle pod labeling for predictable IPs
+	config := designate.PodLabelingConfig{
+		ConfigMapName: designate.BindPredIPConfigMap,
+		IPKeyPrefix:   "bind_address_",
+		ServiceName:   "designate-backendbind9",
+	}
+	err = designate.HandlePodLabeling(ctx, helper, instance.Name, instance.Namespace, config)
```

Comment:
We call this function every reconcile loop, and if I understand correctly, we list all of the pods of the deployment and then filter for the designate ones. We might want to implement a mechanism to call it only when the configmap changes, e.g. comparing hashes as we do with the pools.yaml generation. But we can implement that in a future patch if we see that it is necessary.